### PR TITLE
Add svg_renderer to visual tests

### PR DESCRIPTION
**Pull request by springmeyer**
The svg_renderer has a lot of promise and although it misses many features still I think it's worth starting to test visually to ensure it does not regress and to help test new features.

/cc @gnzzz

### CODE REVIEW

File: test/visual/runner.cpp [hunk 1]
```diff
@@ -61,7 +61,8 @@ runner::runner(runner::path_type const & styles_dir,
       reference_dir_(reference_dir),
       jobs_(jobs),
       renderers_{ renderer<agg_renderer>(output_dir_, reference_dir_, overwrite),
-                  renderer<cairo_renderer>(output_dir_, reference_dir_, overwrite)/*,
+                  renderer<cairo_renderer>(output_dir_, reference_dir_, overwrite),
+                  renderer<svg_renderer>(output_dir_, reference_dir_, overwrite)/*,
```
Comment (talaj) [inline test/visual/runner.cpp]: I forgot to `ifdef` this. If you have time...

Comment (springmeyer) [inline test/visual/runner.cpp]: nice catch, fixed in 018c595

File: test/visual/runner.hpp [hunk 2]
```diff
@@ -36,7 +36,8 @@ namespace visual_tests
 class runner
 {
     using renderer_type = mapnik::util::variant<renderer<agg_renderer>,
-                                                renderer<cairo_renderer>/*,
+                                                renderer<cairo_renderer>,
+                                                renderer<svg_renderer>/*,
```
Comment (talaj) [inline test/visual/runner.hpp]: And here :)

Comment (springmeyer) [inline test/visual/runner.hpp]: 77907bd

**Pull request merged**